src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-sequences */
+/* eslint-disable prettier/prettier */
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    (this.name = name), (this.health = health), Animal.alive.push(this);
```

Comment:
The use of comma operators here is unconventional and can lead to less readable code. It's better to split this into multiple lines for clarity.